helm/values.yaml

```diff
@@ -375,4 +375,38 @@ docling:
   service:
     port: 15001
   imagePullPolicy: IfNotPresent
-  image: "ghcr.io/docling-project/docling-serve-cpu:v0.16.0"
\ No newline at end of file
+  image: "ghcr.io/docling-project/docling-serve-cpu:v0.16.0"
+
+## MCP Context Forge chart configuration
+## ref: https://github.com/IBM/mcp-context-forge/blob/main/charts/mcp-stack/values.yaml
+## @param mcp-stack.enabled Switch to enable or disable the MCP Context Forge helm chart
+##
+mcp-stack:
+  mcpContextForge:
+    replicaCount: 1
+    image:
+      repository: tomaspilar/mcp-context-forge # ghcr.io/ibm/mcp-context-forge
+      tag: latest
+    hpa:
+      enabled: false
+    ingress:
+      enabled: false
+    config:
+      CACHE_TYPE: "memory"
+    secret:
+      AUTH_REQUIRED: "false"      
+  migration:
+    enabled: true
+    image:
+      repository: tomaspilar/mcp-context-forge # ghcr.io/ibm/mcp-context-forge
+      tag: latest
+  postgres:
+    enabled: true
```

Comment:
I rather opted for a separate database since I haven't found configuration for separate schema.

Reply:
let's not have another postgres instance:
- there can be potential naming collisions
- the configuration of postgres in our chart is rather complex and must be substitutable by IBM cloud db (see the section on externalDatabase in values.yaml, we'd need to add another for mcp forge)
- it'll eat away resources for no reason

We need to make schema configurable in context forge if it's not possible at the moment

Reply:
updated to SQLite within the postgres PVC until forge adds support for schemas